minidump-common/src/format.rs

```diff
@@ -6519,8 +6519,22 @@ pub enum ExceptionCodeLinux {
 
 // These values come from asm-generic/siginfo.h
 #[derive(Copy, Clone, PartialEq, Debug, Primitive)]
-pub enum ExceptionCodeLinuxSigillKind {
+#[repr(i32)]
```

Comment:
do you need this if you'r not dealing with ffi?

Reply:
It seems that the representation defaults to u32 so I can't use signed constants for the values (or maybe I'm doing it wrong?).

Reply:
oh, interesting.  TIL